install.sh

```diff
@@ -20,23 +20,41 @@ fi
 #-----------------------
 
 #### Install Packages
-$SUDO apt-get update && $SUDO apt-get install -yq \
```

Comment:
Were these changes necessary? If so, can you explain why?

Reply:
libboost-container-dev is only available on Ubuntu 20.04, not on 18.04, if we try to install all packages at the same time, it would fail and not install any packages. This way it installs the package on 20.04. I needed the xfce4 for my users, since many of them are using virtual machines on weak laptops, you dont need that in general. The other packages were necessary for compilation on bare Ubuntu machines.

Reply:
The existing install.sh script should work fine on 18.04 . We haven't tried 20.04 before, but I ran it through in a docker build today. With minor fixes (in the environment before we run "source ./install.sh", it worked (see the new Dockerfiles in #472). Those Dockerfiles are showing we don't need need these extra packages. I'm starting with the bare ubuntu images from docker hub.